README.md

```diff
@@ -56,69 +40,66 @@ Pizzly is also available as a standalone **OAuth manager**, helping you retrieve
 - JavaScript library to connect from your web-app (three-legged OAuth flow)
 - Provides configurations for over 50+ OAuth APIs (see list below)
 - Support adding new OAuth APIs using a file definition
-- 1-click deploy to Heroku or major cloud hosting solutions
+- 1-click deploy to Heroku or major cloud hosting solutions -->
 
-## Installation
+## Getting started
 
-At the heart of Pizzly is a Node.js application that uses PostgreSQL as a database. It is straightforward to install on Heroku using the **[deploy to Heroku](https://heroku.com/deploy?template=https://github.com/Bearer/Pizzly)** button, but you can install it anywhere.
+Pizzly can be installed anywhere (AWS, Heroku, Platform.sh, etc.). Here's a quick guide using Heroku:
 
-### Manual Install
-
-To run Pizzly on your machine, follow these steps (or [follow our step-by-step guides](https://github.com/Bearer/Pizzly/wiki/Getting-started)):
-
-```bash
-git clone https://github.com/Bearer/Pizzly
-cd pizzly
-yarn install
-yarn db:setup
-yarn start
-```
-
-Then open the dashboard in your browser at:
-
-```
-http://localhost:8080/
-```
+1. First, deploy your own instance of Pizzly by clicking the [Deploy to Heroku](https://heroku.com/deploy?template=https://github.com/Bearer/Pizzly) button below:
 
-You will need Node.js and PostgreSQL installed first. Read our [getting started](https://github.com/Bearer/Pizzly/wiki/Getting-started) to follow a step-by-step installation guide.
+   <a href="https://heroku.com/deploy?template=https://github.com/Bearer/Pizzly" rel="nofollow" target="_blank"><img src="https://www.herokucdn.com/deploy/button.svg" alt="Deploy to Heroku" height="40"></a>
 
-### PaaS Deployment
+2. Once deployed, open your Heroku app. You will land on Pizzly's dashboard.
+3. Click on "Open Dashboard" and select the API you want to integrate with.
+4. Now, configure the API by entering your credentials and scopes. That's all it takes to configure a new API.
+5. To connect a user to this API, _in your frontend_, use the code below:
```

Comment:
It may be worth either:
- Adding the package install details step or
- Adding a link to the package documentation